frontend/src/components/MultiSelection.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+import { MenuItemStatus } from 'types';
+import {
+  HelperText,
+  HelperTextItem,
+  Select,
```

Comment:
Select is being deprecated soon.  We should reconsider this design.

Reply:
Upon writing up the ticket to submit for Lucas to work on -- I am hesitant about looking at different designs if PF hasn't officially declared something deprecated. This would need all new UX design. Going to ignore this comment for the sake of simplicity right now. If PF deprecates something we can look at working with UX to fix it all over our app at the same time.